imu_kvh_1750_mixed.orogen

```diff
@@ -1,11 +1,23 @@
 name "imu_kvh_1750_mixed"
 
+using_library 'iodrivers_base'
 using_task_library "imu_kvh_1750"
 import_types_from "imu_kvh_1750_mixedTypes.hpp"
+import_types_from "base"
 
 task_context "Task" do
```

Comment:
In general, use the generic iodrivers_base integration when adding a iodrivers_base driver in orogen
  http://rock-robotics.org/master/documentation/device_drivers/writing_orogen_driver.html

(Don't change this one, since it will be thrown away)